realm/realm-library/src/main/java/io/realm/internal/ForcedLoadChangeset.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2015 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.realm.internal;
+
+import io.realm.RealmResults;
+
+/**
+ * Fake changeset used if {@link RealmResults#load()} is called manually.
+ */
+public class ForcedLoadChangeset extends OsCollectionChangeSet {
```

Comment:
Why not called `EmptyLoadChangeset` since it's used in this context https://github.com/realm/realm-java/pull/5611/files/ff07d57f17539b9365949465abbda2c85378cb07..65a392c552ce267e726e8fd64ba1f9fb0022740d#diff-013603dffe5ba56b83771eaefb17ce9cR435 
the I feel `ForceLoad` represents the action that triggers the use of this class not it's state